docs/organizations/roles-and-permissions-for-synced-organizations.md

```diff
@@ -189,10 +206,10 @@ Depending on your role on the Git provider you will have different permissions o
 
 See [managing people](managing-people.md) to list and manage the members of your organization.
 
-## Configuring who can ignore issues {: id="configure-ignore-issues"}
+## Configuring who can ignore issues {: id="change-analysis-configuration"}
 
-By default, only users with Write permission on a repository can [ignore issues](../repositories/issues.md#ignoring-and-managing-issues), but you can change this. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to ignore issues for your organization.
+By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues for your organization](../repositories/issues.md#ignoring-and-managing-issues) and [configure your code patterns](../repositories-configure/code-patterns.md).
```

Comment:
Suggestion to clarify that both ignoring issues and configuring code patterns apply at the organization level:

```suggestion
By default, only users with Write permission on a repository can change analysis configurations. In your Organization's settings, you can find **Members privileges** where you can define the lowest permission required to [ignore issues](../repositories/issues.md#ignoring-and-managing-issues) and [configure your code patterns](../repositories-configure/code-patterns.md) on your organization.
```